documentation/release_notes.rst

```diff
@@ -8,6 +8,54 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.3
+=============
+
+New Features
+------------
+-  Added the Range-based versions of the following algorithms: ``all_of``, ``any_of``, ``count``, ``count_if``, ``equal``, ``move``, ``remove``, ``remove_if``, ``replace``, ``replace_if``.
+-  Added the following utility ranges (viewes): ``generate``, ``fill``, ``rotate``.
+
+Changes to Existing Features
+-----------------------------
+-  Improved performance of the ``discard_block_engine`` (including ``ranlux24``, ``ranlux48``, ``ranlux24_vec``, ``ranlux48_vec`` predefined engines) and ``normal_distribution`` RNG functionality.
+- Added two constructors to ``transform_iterator``: ``transform_iterator()`` and ``transform_iterator(iter)``.
```

Comment:
I think, if we change `iter` to `base_iter` in parameters, it would be rather clear. In this case `transform_iterator` constructs transformation functor of type passed in template arguments, **@akukanov**, should we specify it?

Reply:
Changing "iter" to "iter_base" is only marginally better, I think, Try thinking of this note as a concise version of the documentation - what should it minimally say to be properly understood by someone not aware of details?

Reply:
Then let's just explain it in words as you suggested. We can specify that `transform_iterator` constructed this way use transformation functor of type passed in template arguments.

Reply:
I've added discussed changes. Please review